providers/src/airflow/providers/ydb/hooks/ydb.py

```diff
@@ -136,12 +139,11 @@ class YDBHook(DbApiHook):
     supports_autocommit: bool = True
     supports_executemany: bool = True
 
-    def __init__(self, *args, is_ddl: bool = False, use_scan_query: bool = False, **kwargs) -> None:
+    def __init__(self, *args, is_ddl: bool = False, **kwargs) -> None:
```

Comment:
Those all look like breaking changes, can you make a note at the top of ydb Changelog explaining that this is a breaking changes and summarizing changes for the users ?

Then release manager will apply version change when preparing the documentation.